lib/services.js

```diff
@@ -252,8 +225,11 @@ export default {
                     log.warn('error parsing acl', { error: err });
                     return cb(err);
                 }
-                omVal.acl = parsedACL;
-                metadata.putObjectMD(bucketName, objectKey, omVal, log, err => {
+                md.setAcl(parsedACL);
+                log.trace('object metadata', {
+                    omVal: md.getData(),
```

Comment:
I have heard it's better not to store large objects in the logs to minimize performance and storage impact, not sure if object MD qualifies but it's something to consider